satpy/readers/yaml_reader.py

```diff
@@ -1235,3 +1260,47 @@ def _find_missing_segments(file_handlers, ds_info, dsid):
         slice_list.append(None)
 
     return counter, expected_segments, slice_list, failure, projectable
+
+
+def _get_new_area_extent_heights(previous_area, previous_seg_size, segment_n, padding_fci_scene):
+    """Get the area extent heights in meters (projection coordinates) and pixels."""
```

Comment:
Technically the extents can also be scan angle (like ABI), feet, or something else, so maybe just say `in projection coordinates` instead of specifying the unit which isn't enforced in the code in any case.

Reply:
Right, thanks! I'll also rename the function to be `_get_new_areadef_heights`, since the pixel height is not part of the area extent.